src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -48,17 +48,21 @@ function LoadingScreen({
   uploading,
   awaitingSourcemaps,
   message,
-}: PropsFromRedux & { message: string }) {
+  additionalInfo,
+}: PropsFromRedux & { message: string; additionalInfo?: string }) {
   const waitingForMessage =
```

Comment:
Was this an intentional change too? It isn't mentioned in the PR, but now we'll be using the "uploading" message again. (Before, we weren't.)

Reply:
Sorry, I'm not seeing the uploading change you're describing. Looks like the last change on that line was 2 months ago?

Reply:
I'm referring to the `waitingForMessage` change. Before it was being declared but not referenced:

https://github.com/replayio/devtools/blob/856be645dbbe1a0386d4ab1f05cbfae958837d16/src/ui/components/shared/LoadingScreen.tsx#L47-L64

Now it's being referenced:

https://github.com/replayio/devtools/blob/b26e6b013386998f963c9fec3e103936d206a70b/src/ui/components/shared/LoadingScreen.tsx#L53-L68

Reply:
Ah ha, I see it now. Yeah, I think I probably removed that in a previous ticket without realising, then re-stitched it back together in this one.